Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -104,6 +176,16 @@ static void webkit_user_content_manager_class_init(WebKitUserContentManagerClass
             g_cclosure_marshal_VOID__BOXED,
             G_TYPE_NONE, 1,
             WEBKIT_TYPE_JAVASCRIPT_RESULT);
+
+    signals[SCRIPT_REQUEST_RECEIVED] =
+        g_signal_new(
+            "script--received",
```

Comment:
I suppose this is a typo, there are two dashes in the signal name and we had agred to call the new signal `script-request-received`... probably you removed the “request” word accidentally 😉 

```suggestion
            "script-request-received",
```

Reply:
My $0.02: the `script-message` terminology is slightly nicer than `script-request`.